### PR TITLE
enhnacement to the ARCHITECTURE.md file

### LINKED ISSUE #449: Enhancement: Make ARCHITECTURE.md easier to read
The sections for "Regenerate Jobs" and "Pipeline Jobs" have run-on sentences that tie my brain in knots. 

e.g. "These are the initial scripts that are the top level jobs that are invoked, etc etc"

I propose we break the sentences down into smaller sentences which are clearer.

e.g. "These are the initial scripts that are the top level jobs that are invoked," could be "These are the first scripts run by the top level jobs,", or similar. 

As a rule, this task is only complete once each sentence has two commas or less, and minimal redundancy.

### PR DESCRIPTION
**Pull request by lumuchris256**
enhnacement to the ARCHITECTURE.md file
fixes #449 
signed off by ; @lumuchris256
cc @adamfarley

### CODE REVIEW

File: docs/ARCHITECTURE.md [hunk 1]
```diff
@@ -16,15 +16,12 @@ A rough structure of the build is as follows
 
 #### Regenerate Jobs
 
-Jenkins CI jobs are generated by the [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/) in Jenkins. The documentation for this process can be found at [regeneration docs](https://github.com/adoptium/temurin-build/tree/master/pipelines/build/regeneration), however they generate the jobs
-at <https://ci.adoptopenjdk.net/job/build-scripts/> which are the builds that coordinate the entire process of nightly
-and release builds.
+Jenkins CI jobs are generated by the Jenkins [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/). Generated at <https://ci.adoptopenjdk.net/job/build-scripts/> and their builds coordinate the entire process of nightly and release builds.
```
Comment (karianna) [inline docs/ARCHITECTURE.md:19]: I'm still not clear on what the second sentence is telling me.

Comment (lumuchris256) [reply docs/ARCHITECTURE.md:19]: @karianna 
would rather say "and their builds populate the nightly and release builds "   ?

Comment (karianna) [reply docs/ARCHITECTURE.md:19]: That's better yes, I also just realised what the "Generated at <https://ci.adoptopenjdk.net/job/build-scripts/>" actually means, that's the location of where the generated jobs are created.  I'll make a proposed commit.

Comment (karianna) [inline docs/ARCHITECTURE.md:19]:
```suggestion
Jenkins CI jobs are generated by the Jenkins [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/) and placed into the <https://ci.adoptopenjdk.net/job/build-scripts/> location. These builds coordinate the entire process of nightly and release builds.
```

**Review by karianna — APPROVED** [review]

**Review by zdtsw — APPROVED** [review]

**Pull request merged**